assets/stylesheets/utilities/_utilities.scss

```diff
@@ -67,6 +67,10 @@ a.underline,
   text-decoration: underline;
 }
 
+.text-shadow-75 {
```

Comment:
@zarahzachz Are you sure you want to change the convention from what you have in hotfix? If so, you'll need to consider the implications if this decision, post prod deploy.